client/styles/components/component_kit/cards.scss

```diff
@@ -0,0 +1,58 @@
+@import 'client/styles/facelift';
+
+$elevation-1-shadow: 0px 4px 12px rgba(0, 0, 0, 0.05);
```

Comment:
I feel like we should store these css variables in the facelift style file we import

Reply:
if we plan on using any of these in other files!